### PR TITLE
enable the country data report to plot more parameter values that hav…

### PR DESCRIPTION
**Pull request by KaiyueZou**
…e been estimated

### CODE REVIEW

File: packages/taxdat/R/plot_MCMC.R [hunk 1]
```diff
@@ -27,10 +27,12 @@ plot_chain_convergence <- function(name,
   
   time_effect_param <- tolower(yaml::read_yaml(paste0(cholera_directory,"/",config))$time_effect)
   if(time_effect_param == 'no' | time_effect_param == 'false'){
-    pars <- pars[pars != "eta"]
+    warning("There is no time effect, meaning eta and alpha will not be shown. ")
```
Comment (eclee25) [inline packages/taxdat/R/plot_MCMC.R:30]: There may still be an alpha even if the time_effect is off. We specify whether an intercept should be in the model separately

File: packages/taxdat/R/plot_MCMC.R [hunk 2]
```diff
@@ -45,14 +47,20 @@ plot_chain_convergence <- function(name,
...
+                          cache, 
+                          pars = c("rho", "alpha", "log_std_dev_w", "eta", "std_dev_w", "od_param"),
+                          config, 
+                          cholera_directory) {
+  model.rand <- cache[[name]]
+  
   time_effect_param <- tolower(yaml::read_yaml(paste0(cholera_directory,"/",config))$time_effect)
   if(time_effect_param == 'no' | time_effect_param == 'false'){
-    pars <- pars[pars != "eta"]
+    warning("There is no time effect, meaning eta and alpha will not be shown. ")
```
Comment (eclee25) [inline packages/taxdat/R/plot_MCMC.R:59]: same comment as above

**Review by eclee25 — CHANGES_REQUESTED** [review]
For the trace plot functions, rather than checking if each parameter exists based on the config options, is it possible to have a more generic function that tries to plot all parameters from a pre-specified list and then skips the ones that don't exist?

**Comment by KaiyueZou** [comment]
> ove works for me. However

I see, I will change the warning message

**Pull request merged**